### PR TITLE
feat(WIP): playground

### PR DESCRIPTION
**Pull request by distractedm1nd**
TODO:
- [x] Fix Notifications
- [x] Configuration section (URL, auth token)
- [x] Merge CORS fix in node
- [x] Styling improvements (i dont really like the modal yet, and seems too jumpy. might add transitions)

https://github.com/celestiaorg/node-rpc-docs/assets/16523232/5541ed6a-2f5e-4f78-a628-5a4012b59fd9

### CODE REVIEW

**Comment by jcstein** [comment]
I noticed while reviewing that if in medium and smaller screen sizes, the playground modal disappears. Though, the CI was green until I resolved conflicts in index.tsx at https://github.com/celestiaorg/node-rpc-docs/pull/45/commits/91d585c8fb4e437d97ab293b86580a67e9558114, but I think this is because of formatting.

Screenshot 2024-06-17 at 12 57 45 PM

File: src/components/ExampleTypeModal.tsx
[the remark this answers is not in the document]
Comment (jcstein) [reply src/components/ExampleTypeModal.tsx:1]: https://github.com/celestiaorg/node-rpc-docs/pull/45/commits/e07580c20598c78131f373d764dfd07e01e12e18

**Comment by jcstein** [comment]
Screenshot 2024-06-17 at 1 30 15 PM
https://github.com/celestiaorg/node-rpc-docs/pull/45/commits/f144f63be3132e1ec6e4ca76e69a9ff5de914d6a

**Review by jcstein — APPROVED** [review]
LGTM. are all the things in todo list done? if not we can move unfinished tasks to new issues

**Comment by distractedm1nd** [comment]
Thanks for fixing the rest!! Looks good to me. The TODOs are complete

**Pull request merged**